.github/workflows/test-and-release.yml

```diff
@@ -57,9 +57,8 @@ jobs:
       - test-simple
     strategy:
       matrix:
-        go-version: ["1.17", "latest"]
-        terramate-version: ["v0.4.2", "latest"]
-        terraform-version: ["1.5.7", "latest"]
+        terramate-version: ["latest"]
+        terraform-version: ["latest"]
```

Comment:
Test other versions than latest?

Reply:
which one? then we have to maintain that info... not sure...

Reply:
before we had
```yaml
terramate-version: ["v0.4.2", "latest"]
terraform-version: ["1.5.7", "latest"]
```